src/main/java/de/zalando/aruha/nakadi/domain/Subscription.java

```diff
@@ -10,9 +10,15 @@
 import java.util.Set;
 
 import static com.google.common.collect.Sets.newTreeSet;
+import static de.zalando.aruha.nakadi.domain.Subscription.InitialPosition.END;
 
 public class Subscription {
 
+    public enum InitialPosition {
+        BEGIN,
+        END
+    }
```

Comment:
According to current implementation, cursor can have offset named 'BEGIN' (with capital letters)
And this one will be in lowercase, because it's enumeration. I think that we need to make it the same way in both cases.